.github/actions/validate-with-schema/action.yml

```
@@ -0,0 +1,46 @@
+name: Validate With Schema
+description: Action that validates json/yaml data against a given schema
+inputs:
+  schema-version:
+    type: string
+    required: true
+    description: Version of the schema required, in SchemaVer
+  schema-location:
+    type: string
+    required: true
+    description: Directory within access-nri/schema that contains the schema
+  data-location:
+    type: string
+    required: true
+    description: Path to the data in the callers repository
+runs:
+  using: composite
+  steps:
+      # Checkout the callers `spack.yaml`
```

```suggestion
      # Checkout the callers repository to access the data to be checked
```

I guess this comment is left over from where it was copied from?

---

Done!